types/dasha.d.ts

```diff
@@ -145,8 +145,7 @@ export type SubtitleCodec =
   | 'TTML'
   | 'VTT'
   | 'STPP'
-  | 'fTTML'
-  | 'fVTT';
```

Comment:
Because I think what is returned here is the **value**, **not** the **key**, so there will be no `fTTML` and `fVTT`.

https://github.com/vitalygashkov/dasha/blob/05cdf4039c758fda80e364e14d73d6837351a1b8/lib/subtitle.js#L6C1-L38C3
```js
const SUBTITLE_CODECS = {
  // ...
  fTTML: 'STPP', // https://www.w3.org/TR/2018/REC-ttml-imsc1.0.1-20180424
  fVTT: 'WVTT', // https://www.w3.org/TR/webvtt1
};

const parseSubtitleCodecFromMime = (mime) => {
  const target = mime.toLowerCase().trim().split('.')[0];
  switch (target) {
    // ...
    case 'stpp':
      return SUBTITLE_CODECS.fTTML; // <- return the value, not the key
    case 'wvtt':
      return SUBTITLE_CODECS.fVTT;
    // ...
  }
};
```